ts/features/wallet/onboarding/bancomat/saga/networking/index.ts

```diff
@@ -93,3 +94,35 @@ export function* handleLoadPans(
     return yield put(searchUserPans.failure({ kind: "generic", value: e }));
   }
 }
+
+// add pan to wallet
+export function* handleAddPan(
+  addPans: ReturnType<typeof PaymentManagerClient>["addPans"],
+  sessionManager: SessionManager<PaymentManagerToken>,
+  action: ActionType<typeof addWalletBancomatPan.request>
+) {
+  try {
+    const addPansWithRefresh = sessionManager.withRefresh(
+      // add a card as an array of one element
+      addPans({ data: [action.payload] })
+    );
+    const addPansWithRefreshResult: SagaCallReturnType<typeof addPansWithRefresh> = yield call(
+      addPansWithRefresh
+    );
+    if (addPansWithRefreshResult.isRight()) {
+      if (addPansWithRefreshResult.value.status === 200) {
+        yield put(
+          addWalletBancomatPan.success(addPansWithRefreshResult.value.value)
+        );
```

Comment:
Can we return the single `WalletV2` received? 
In our case we call the `addWalletBancomatPan.request` with a single `Card` so in case of `addWalletBancomatPan.success` is expected to have a single `WalletV2`.

Reply:
Fixed within https://github.com/pagopa/io-app/pull/2286/commits/c1b0f92ada512af8767c69e70c72bf345d0ae398